engine/gamesys/src/gamesys/test/resource/create_texture.script

```diff
@@ -60,6 +74,28 @@ function test_release_not_found()
 	end)
 end
 
+function test_compressed()
+	local f     = io.open("src/gamesys/test/resource/blank.basis", "rb")
```

Comment:
Have you tested this on ps4/switch? Since I don't think it will work as-is.

Reply:
I'm using the "WrapIoFunctions" in test_gamesys, which should work.